install/kubernetes/helm/istio/values.yaml

```diff
@@ -8,19 +8,22 @@ global:
   # Default repository for Istio images.
   # Releases are published to docker hub under 'istio' project.
   # Daily builds from prow are on gcr.io, and nightly builds from circle on
-  # docker.io/istionightly
-  hub: docker.io/istio
+  hub: docker.io/istionightly
+  #hub: docker.io/istio
 
   # Default tag for Istio images.
-  # Should track latest released version in the branch.
-  tag: 0.8.latest
+  tag: nightly-master
+  #tag: 1.0.latest
```

Comment:
is this leftover?

Reply:
No, in the branch it'll have a different value. We want each branch to default to latest nightly build ( or daily - if they start getting stable tags ). This way someone can use the templates and test the release - without setting a tag override or building.